lib/_stream_readable.js

```diff
@@ -678,6 +678,17 @@ Readable.prototype.on = function(ev, fn) {
 };
 Readable.prototype.addListener = Readable.prototype.on;
 
+Readable.prototype.removeListener = function(type, listener) {
+  Stream.prototype.removeListener.call(this, type, listener);
```

Comment:
It would trigger an infinite loop; `this.removeListener` is the function we're defining. I used the same syntax as that of `Readable.prototype.on` to call the prototype's version of the method.